DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/Bindings/ArcGISSendBinding.cs

```diff
@@ -87,77 +73,30 @@ public List<CardSetting> GetSendSettings()
 
       // 1 - Get model
-
       if (_store.GetModelById(modelCardId) is not SenderModelCard modelCard)
       {
         throw new InvalidOperationException("No publish model card was found.");
       }
 
-      // 2 - Check account exist
-      // TODO: FETCHING ACCOUNTS BY ID ONLY IS UNSAFE
-      Account account =
-        AccountManager.GetAccounts().FirstOrDefault(acc => acc.id == modelCard.AccountId)
-        ?? throw new SpeckleAccountManagerException();
-
-      // 3 - Get elements to convert, throw early if nothing is selected: TODO
-      List<string> arcgisObjects = new() { "object1", "object2" };
-
-      if (arcgisObjects.Count == 0)
-      {
-        throw new InvalidOperationException("No objects were found. Please update your send filter!");
-      }
-
-      var converter = _speckleConverterToSpeckleFactory.ResolveScopedInstance();
-      // 5 - Convert objects
-      Base commitObject = ConvertObjects(arcgisObjects, converter, modelCard, cts);
-
-      if (cts.IsCancellationRequested)
-      {
-        throw new OperationCanceledException(cts.Token);
-      }
-
-      // 7 - Serialize and Send objects
-      Commands.SetModelProgress(modelCardId, new ModelCardProgress { Status = "Uploading..." });
-
-      var transport = new ServerTransport(account, modelCard.ProjectId);
-      var sendResult = await SendHelper.Send(commitObject, transport, true, null, cts.Token).ConfigureAwait(true);
-
-      // Store the converted references in memory for future send operations, overwriting the existing values for the given application id.
-      foreach (var kvp in sendResult.convertedReferences)
-      {
-        // TODO: Bug in here, we need to encapsulate cache not only by app id, but also by project id,
-        // TODO: as otherwise we assume incorrectly that an object exists for a given project (e.g, send box to project 1, send same unchanged box to project 2)
-        _convertedObjectReferences[kvp.Key + modelCard.ProjectId] = kvp.Value;
-      }
-      // It's important to reset the model card's list of changed obj ids so as to ensure we accurately keep track of changes between send operations.
-      // NOTE: ChangedObjectIds is currently JsonIgnored, but could actually be useful for highlighting changes in host app.
-      //modelCard.ChangedObjectIds = new();
-
-      Commands.SetModelProgress(modelCardId, new ModelCardProgress { Status = "Linking version to model..." });
-
-      // 8 - Create the version (commit)
-      var apiClient = new Client(account);
-      string versionId = await apiClient
-        .CommitCreate(
-          new CommitCreateInput
-          {
-            streamId = modelCard.ProjectId,
-            branchName = modelCard.ModelId,
-            sourceApplication = "ArcGIS",
-            objectId = sendResult.rootObjId
-          },
+      string versionId = await _sendOperation
+        .Execute(
+          //modelCard.SendFilter,
```

Comment:
Right now, no because I wanted to unblock kat while I am away, and it wasn't obvious how to get the send filter to work with the arcgis api (which works differently from Rhino). more investigation required on this.